port/zephyr/Kconfig

```diff
@@ -59,8 +59,9 @@ config GOLIOTH_AUTH_PSK_MBEDTLS_DEPS
 config GOLIOTH_AUTH_CERT_MBEDTLS_DEPS
 	bool "mbedTLS dependencies for certificate based auth"
 	depends on MBEDTLS_BUILTIN || NRF_SECURITY
-	# At least one ECP needs to be selected. This was chosen arbitrarily.
+	# Leaf certs use P-256, Intermediate and Root certs use P-384
```

Comment:
nitpick: @sam-golioth can we say something like `Enable support for both P-256 and P-384 curves for broad compatibility` or similar here rather than tying this description to the current trust chain on the server?